secure/configuring-security-context-for-pods.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="configuring-security-context-for-pods"]
+= Configuring the security context for pods
+:context: configuring-security-context-for-pods
+
+toc::[]
+
+The default service account for pods that {pipeline-shortname} starts is `pipeline`. The security context constraint (SCC) associated with the `pipeline` service account is `pipelines-scc`. The `pipelines-scc` SCC is based the `anyuid` SCC, with minor differences as defined in the following YAML specification:
+
+.Example `pipelines-scc.yaml` snippet
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+# ...
+allowedCapabilities:
+  - SETFCAP
+# ...
+fsGroup:
+  type: MustRunAs
+# ...
+----
+
+In addition, the `Buildah` cluster task, shipped as part of {pipelines-shortname}, uses `vfs` as the default storage driver.
+
+You can configure the security context for pods that {pipelines-shortname} creates for pipeline runs and task runs. You can make the following changes:
+
+* Change the default and maximum SCC for all pods
+* Change the default SCC for pods created for pipeline runs and task runs in a particular namespace
+* Configure a particular pipeline run or task run to use a custom SCC and service account
+
+[NOTE]
+====
+The simplest way to run `buildah` that ensures all images can build is to run it as root in a pod with the `privileged` SCC. For instructions about running `buildah` with more restrictive security settings, see xref:../secure/unprivileged-building-of-container-images-using-buildah.adoc#unprivileged-building-of-container-images-using-buildah[Building of container images using Buildah as a non-root user].
+====
+
+include::modules/op-configuring-default-maximum-scc.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../install_config/customizing-configurations-in-the-tektonconfig-cr.adoc#op-changing-default-service-account_customizing-configurations-in-the-tektonconfig-cr[Changing the default service account for OpenShift Pipelines]
+
+include::modules/op-configuring-scc-namespace.adoc[leveloffset=+1]
+
+include::modules/op-running-pipeline-run-and-task-run-with-custom-scc-and-service-account.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-references_configuring-security-context-for-pods"]
+== Additional resources
+
+* For information about the available SCCs and managing SCCs, refer to link:https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html[Managing security context constraints].
```

Comment:
Small nit: IIUC, "Additional resources" sections are supposed to only include the link and the human-readable text, not additional text around them. I would change this to simply:

* link:https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html[Managing security context constraints]